examples/release_highlights/plot_release_highlights_0_24_0.py

```diff
@@ -0,0 +1,236 @@
+# :class:`~sklearn.feature_selection.SequentialFeatureSelector`.
+# Sequential Feature Selection can add features once at a time (forward
+# selection) or remove features from the list of the available features
+# (backward selection), based on a cross-validated score maximization.
+# See the :ref:`User Guide <sequential_feature_selection>`.
+
+from sklearn.feature_selection import SequentialFeatureSelector
+from sklearn.neighbors import KNeighborsClassifier
+from sklearn.datasets import load_iris
+
+X, y = load_iris(return_X_y=True, as_frame=True)
+feature_names = X.columns
+knn = KNeighborsClassifier(n_neighbors=3)
+sfs = SequentialFeatureSelector(knn, n_features_to_select=2)
+sfs.fit(X, y)
+print("Features selected by forward sequential selection: "
+      f"{feature_names[sfs.get_support()]}")
+
+##############################################################################
+# New PolynomialCountSketch kernel approximation function
+# -------------------------------------------------------
+# The new :class:`~sklearn.kernel_approximation.PolynomialCountSketch`
+# approximates a polynomial expansion of a feature space when used with linear
+# models, but uses much less memory than
+# :class:`~sklearn.preprocessing.PolynomialFeatures`.
+
+from sklearn.datasets import fetch_covtype
+from sklearn.pipeline import make_pipeline
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import MinMaxScaler
+from sklearn.kernel_approximation import PolynomialCountSketch
+from sklearn.linear_model import LogisticRegression
+
+X, y = fetch_covtype(return_X_y=True)
+pipe = make_pipeline(MinMaxScaler(),
+                     PolynomialCountSketch(degree=2, n_components=100),
+                     LogisticRegression(max_iter=1000))
+X_train, X_test, y_train, y_test = train_test_split(X, y, train_size=5000,
+                                                    test_size=10000,
+                                                    random_state=42)
+pipe.fit(X_train, y_train).score(X_test, y_test)
+
+linear_baseline = make_pipeline(MinMaxScaler(),
+                                LogisticRegression(max_iter=1000))
+linear_baseline.fit(X_train, y_train).score(X_test, y_test)
+
+##############################################################################
+# Individual Conditional Expectation
+# ----------------------------------
+# A new kind of partial dependence plot is available: the Individual
+# Conditional Expectation (ICE) plot. It visualizes the dependence of the
+# prediction on a feature for each sample separately with one line per sample.
+# See the :ref:`User Guide <individual_conditional>`
+
+from sklearn.linear_model import BayesianRidge
+from sklearn.datasets import fetch_california_housing
+from sklearn.inspection import plot_partial_dependence
+
+X, y = fetch_california_housing(return_X_y=True, as_frame=True)
+features = ['MedInc', 'AveOccup', 'HouseAge', 'AveRooms']
+est = BayesianRidge()
```

Comment:
The `BayesianRidge` is a linear model and therefore the resulting PDP / ICE plots degenerate to boring straight lines with the slope being of the coefficient of the fitted linear model.

I think this example would be more interesting with a non-linear model, e.g. a `RandomForestRegressor()` (assuming it's not too slow to train).

Reply:
If it is too slow we can reduce the number `grid_resolution=25`.